packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -420,56 +426,50 @@ export default class extends EntityModel {
     );
 
     await this.createOutgoingLink(graphApi, {
-      linkTypeModel: SYSTEM_TYPES.linkType.hasMembership,
-      targetEntityModel: orgMembership,
+      linkEntityTypeModel: SYSTEM_TYPES.linkEntityType.hasMembership,
+      rightEntityModel: orgMembership,
       ownedById: systemAccountId,
       actorId,
     });
   }
 
   async getOrgMemberships(graphApi: GraphApi): Promise<OrgMembershipModel[]> {
-    const { data: outgoingOrgMembershipLinkRootedSubgraphs } =
-      await graphApi.getLinksByQuery({
-        filter: {
-          all: [
-            {
-              equal: [{ path: ["source", "id"] }, { parameter: this.entityId }],
-            },
-            {
-              equal: [
-                { path: ["type", "versionedUri"] },
-                {
-                  parameter: SYSTEM_TYPES.linkType.hasMembership.schema.$id,
-                },
-              ],
-            },
-          ],
-        },
-        graphResolveDepths: {
-          dataTypeResolveDepth: 0,
-          propertyTypeResolveDepth: 0,
-          linkTypeResolveDepth: 0,
-          entityTypeResolveDepth: 0,
-          linkResolveDepth: 0,
-          linkTargetEntityResolveDepth: 0,
-        },
-      });
-
-    return await Promise.all(
-      outgoingOrgMembershipLinkRootedSubgraphs.map(async ({ link }) => {
-        const orgMembership = await OrgMembershipModel.getOrgMembershipById(
-          graphApi,
+    const outgoingOrgMembershipLinkEntityModels =
+      await LinkEntityModel.getByQuery(graphApi, {
+        all: [
           {
-            entityId: link.inner.targetEntityId,
+            equal: [
+              { path: ["leftEntity", "uuid"] },
+              { parameter: this.entityUuid },
+            ],
           },
-        );
-
-        if (!orgMembership) {
-          throw new Error("Critical: could not find target of link");
-        }
+          {
+            equal: [
+              { path: ["leftEntity", "ownedById"] },
+              { parameter: this.ownedById },
+            ],
+          },
+          {
+            equal: [
+              { path: ["type", "versionedUri"] },
+              {
+                parameter: SYSTEM_TYPES.linkEntityType.hasMembership.schema.$id,
+              },
+            ],
+          },
+          {
+            equal: [{ path: ["version"] }, { parameter: "latest" }],
+          },
+          {
+            equal: [{ path: ["archived"] }, { parameter: false }],
```

Comment:
Are we able to use the `EntityModel`'s link helper methods? It seems that `getOutgoingLinks` would do here with the type param set